libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -259,9 +259,7 @@ float AP_MotorsMulticopter::get_current_limit_max_throttle()
     }
 
     // calculate the maximum current to prevent voltage sag below _batt_voltage_min
-    float batt_current_max = MIN(_batt_current_max, _batt_current + (_batt_voltage-_batt_voltage_min)/_batt_resistance);
```

Comment:
I believe it is correct in that it reflects the intent of this PR - which is to remove protection against voltage drop below MOT_BAT_VOLT_MIN.

You are correct that the comment needs to be removed.

I think the alternative would be to replace it with something like:
```
float batt_current_max = MIN(_batt_current_max, _batt_current + MAX(_batt_voltage-_batt_voltage_min, 0)/_batt_resistance);
```